extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -516,19 +520,22 @@ export class AmpAnalytics extends AMP.BaseElement {
     }
 
     const request = this.requests_[requestName];
+    const hasPostMessage = this.isInabox_ && trigger['parentPostMessage'];
 
-    if (!request) {
+    if (requestName != undefined && !request) {
       const TAG = this.getName_();
       this.user().error(TAG, 'Ignoring event. Request string ' +
           'not found: ', trigger['request']);
```

Comment:
Weird to say we're ignoring the event if we might not be?

Reply:
Rephrase to `ignoring request for event. Request .....` Let me know if that's better.